doc/design/dist/submit-job.md

```diff
@@ -0,0 +1,70 @@
+
+# PaddlePaddle Client
```

Comment:
Since you are describing the PaddlePaddle client, do you need to describe the full features of it, like local training, show version etc.